android/app/BUCK

```diff
@@ -6,7 +6,7 @@ import re
 # - `npm start` - to start the packager
 # - `cd android`
 # - `cp ~/.android/debug.keystore keystores/debug.keystore`
-# - `./gradlew :app:copyDownloadableDepsToLibs` - make all Gradle compile dependencies available to Buck
+# - `gradle :app:copyDownloadableDepsToLibs` - make all Gradle compile dependencies available to Buck
```

Comment:
that looks like an RN upgrade/downgrade

Reply:
I'm not sure I understand the comment. I changed all calls to gradlew to instead use Nix's gradle executable.

Reply:
ah, okay, just together with comments that changed, I thought that it was suspicious :)
I didn't link it to status-go, tbh.